example/linux/Makefile

```diff
@@ -103,7 +107,6 @@ $(WRAPPER_SOURCES) $(FLUTTER_LIB) $(ICU_DATA_SOURCE): $(FLUTTER_COPY_STAMP_FILE)
 # When the Flutter engine version changes, the local copy of engine artifacts
 # needs to be updated.
 $(FLUTTER_COPY_STAMP_FILE): $(FLUTTER_ENGINE_VERSION_FILE)
-	$(FLUTTER_BIN) precache --linux --no-android --no-ios
```

Comment:
Removing this may break non-`flutter run` cases; we should probably leave it until the copy is being done from the Flutter side.